# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296506283 28800
#      Mon Jan 31 12:38:03 2011 -0800
# Node ID 5cb9a7e08f6697d98d494b8491d5a2a5a3ba3213
# Parent  7b12892ae589e25754fa8e57f765705ed91a3250
Simplify regexp to make it POSIX compliant.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-01-31  Rik  <octave@nomad.inbox5.com>
+
+	* plot/private/__gnuplot_has_terminal__.m: Simplify regular expression
+	to eliminate word-boundary assertion.
+
 2010-01-30  Rik  <octave@nomad.inbox5.com>
 
 	* deprecated/module.mk, optimization/module.mk: Deprecate glpkmex
 	function.
 
 2011-01-30  John W. Eaton  <jwe@octave.org>
 
 	* plot/__gnuplot_drawnow__.m: Rename from gnuplot_drawnow.m
diff --git a/scripts/plot/private/__gnuplot_has_terminal__.m b/scripts/plot/private/__gnuplot_has_terminal__.m
--- a/scripts/plot/private/__gnuplot_has_terminal__.m
+++ b/scripts/plot/private/__gnuplot_has_terminal__.m
@@ -30,17 +30,17 @@ function gnuplot_supports_term = __gnupl
   if (! isempty (n))
     term = term(1:n-1);
   endif
   if (__gnuplot_has_feature__ ("variable_GPVAL_TERMINALS"))
     if (nargin < 2)
       plot_stream = __gnuplot_open_stream__ (2);
     endif
     available_terminals = __gnuplot_get_var__ (plot_stream, "GPVAL_TERMINALS");
-    available_terminals = regexp (available_terminals, '\b\w+\b', "match");
+    available_terminals = regexp (available_terminals, '\w+', "match");
     if (nargin < 2 && ! isempty (plot_stream))
       pclose (plot_stream(1));
       if (numel (plot_stream) > 1)
         pclose (plot_stream(2));
       endif
       if (numel (plot_stream) > 2)
         waitpid (plot_stream(3));
       endif
